pyproject.toml

```diff
@@ -67,6 +67,7 @@ testpaths = ["tests"]
 norecursedirs = ["tests/non-pytest"]
 filterwarnings = [
     "error",
+    "ignore:'SearchQuery'*:DeprecationWarning",
```

Comment:
Max and I attempted to use the `pytest.mark.filterwarnings` decorator but it doesn't work with classes that are being instantiated within pytest.mark.parametrize. I felt it was best to add the ignore to the `pyproject.toml` for now until the legacy SearchQuery is fully deprecated and then it could be removed.